qemu/tests/block_copy.py

```diff
@@ -46,6 +46,9 @@ def __init__(self, test, params, env, tag):
         self.data_dir = data_dir.get_data_dir()
         self.device = self.get_device()
         self.image_file = self.get_image_file()
+        self.params["monitor_type"] = "qmp"
+        self.params["monitors"] = "qmp1"
```

Comment:
better to more monitors,  one for check job status, another one to send  block job commands. so that we avoid monitor   lock in multi-threads.